src/containers/upcoming-events/UpcomingEvents.tsx

```diff
@@ -38,10 +51,18 @@ const Events: React.FC = () => {
             <Radio.Button value="calendar">Calendar</Radio.Button>
           </StyledRadio>
         </Content>
-        <EventsList></EventsList>
+        {events.kind === AsyncRequestKinds.Completed && (
+          <EventsList events={events.result} />
+        )}
```

Comment:
Not really blocking merging this in, but shouldn't we display that the page is loading while it fetches upcoming events?